packages/mobile/src/core/NFTSend/types.ts

```diff
@@ -0,0 +1,4 @@
+export enum NFTSendSteps {
+  ADDRESS = 0,
+  CONFIRM = 1,
```

Comment:
you can just leave it as 
```suggestion
  ADDRESS,
  CONFIRM,
```